src/software/new_geom/line.cpp

```diff
@@ -2,12 +2,21 @@
 
 Line::Line(const Point &first, const Point &second)
 {
+    if (first == second)
+    {
+        throw std::runtime_error("Cannot create a Line with two equal Points");
+    }
     coeffs[0] = first.y() - second.y();
     coeffs[1] = second.x() - first.x();
     coeffs[2] = first.x() * second.y() - second.x() * first.y();
 }
 
-Vector Line::toNormalUnitVector()
+Line::Coeffs Line::getCoeffs() const
+{
+    return {coeffs[0], coeffs[1], coeffs[2]};
```

Comment:
Could we change this to:
```
Coeffs returnme;
returnme.a = coeffs[0];
etc.
```
(also with a better name for `returnme`). If we were ever to change the order of the struct members, what is here currently would still compile :grimacing:

Reply:
Not really any reason not to, changed the internal representation to struct